slack/config.go

```diff
@@ -2,9 +2,30 @@ package slack
 
 // SlackConfig holds configuration for sending Slack notifications
 type SlackConfig struct {
-	APIToken string `envconfig:"SLACK_API_TOKEN"`
-	Username string `envconfig:"SLACK_USERNAME"`
-	Channel  string `envconfig:"SLACK_CHANNEL"`
-	Emoji    string `envconfig:"SLACK_EMOJI"`
-	Enabled  bool   `envconfig:"SLACK_ENABLED"`
+	Channels Channels
+	Enabled  bool `envconfig:"SLACK_ENABLED"`
```

Comment:
As mentioned in a previous comment, this value might be better placed at a higher level in the config file.